index.html

```diff
@@ -250,6 +252,7 @@ <h3 class="text-center">All charts</h3>
 <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.4/lodash.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.16.2/axios.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.min.js"></script>
+<script src="https://unpkg.com/vue-konami"></script>
```

Comment:
Pin the lib version here too like on the other tags, it's risky always loading the latest version.